#goal definition
TeachCommand[] cmd_list
---
#result definition
string notify
---
#feedback
string status
